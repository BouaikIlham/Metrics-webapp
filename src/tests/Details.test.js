import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import Details from '../Components/Details';
import store from '../redux/configStore';

describe('Details tests', () => {
    test('should render', () => {
        expect(Details).toMatchSnapshot();
    });

    test('Checks if details content is there by checking text', () => {
        render(
            <Provider store={store}>
                <BrowserRouter>
                    <Details />
                </BrowserRouter>
                ,
            </Provider>
        );

        expect(
            screen.getByRole('img', { name: /Covid-19 img/i })
        ).toBeInTheDocument();
    });
});