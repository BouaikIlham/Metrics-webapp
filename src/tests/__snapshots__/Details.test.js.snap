// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details tests should render 1`] = `[Function]`;
